packages/fether-react/src/stores/createAccountStore.spec.js

```diff
@@ -26,39 +26,70 @@ beforeEach(() => {
   createAccountStore = new CreateAccountStore();
 });
 
+describe('method generateNewAccount', () => {
+  test('should generate a valid bip39 phrase and derive the address', async () => {
+    await createAccountStore.generateNewAccount();
+
+    expect(bip39.validateMnemonic(createAccountStore.bip39Phrase)).toBe(true);
+
+    expect(createAccountStore.address).toMatch(/^0x[a-fA-F0-9]{40}$/);
+  });
+});
+
 describe('method clear', () => {
-  test('should reset phrase, address and name', () => {
-    createAccountStore.phrase = 'phrase';
+  test('should reset phrase, address and name', async () => {
+    createAccountStore.jsonString = '{}';
+    createAccountStore.parityPhrase = 'phrase';
+    createAccountStore.bip39Phrase = 'phrase';
     createAccountStore.address = '0x123';
     createAccountStore.name = 'account name';
-    createAccountStore.clear();
-    expect(createAccountStore.phrase).toBe(null);
+    await createAccountStore.clear();
```

Comment:
Is await needed here?

Reply:
ah, you're right, createAccountStore.clear doesn't return a promise. I'll make it `async` for consistency